README.md

```diff
@@ -2,6 +2,11 @@
 Learning version control with git
 
 This is currently under development.
-Created to teach people about version control using git.
+Created to teach people about version control using git and GitHub. Feel free to use it for your own purpose, by referring back to the original contributors </n>
```

Comment:
Remove sentence, `Feel free to use it for your own purpose, by referring back to the original contributors </n>`

The terms are in he license.

Reply:
Also, remove Github. The main focus is on teaching git.

Reply:
Removed.